pages/library/_viewId.vue

```diff
@@ -26,6 +26,17 @@
         :items-type="viewType"
         @change="onChangeFilter"
       />
+      <v-divider inset vertical class="mx-2 hidden-sm-and-down" />
```

Comment:
```suggestion
      <v-spacer />
```

This aligns the playback buttons to the right, keeping them consistent with the Genre pages.